src/ir/ty.rs

```diff
@@ -104,9 +104,13 @@ impl Type {
         Self::new(Some(name), None, kind, false)
     }
 
-    pub fn is_integer_literal(&self) -> bool {
+    pub fn is_integer(&self, ctx: &BindgenContext) -> bool {
         match *self.kind() {
             TypeKind::Int(..) => true,
+            TypeKind::ResolvedTypeRef(inner) |
+            TypeKind::Alias(_, inner) => {
```

Comment:
Yeah, if you have llvm 3.9 `llvm_stable` will fail, that's known. If you take care of uploading the test I can update the expectations before merging myself when I arrive home.